src/sheldon_dictionary.erl

```diff
@@ -24,7 +24,7 @@
 
 %% API
 -export([ start_link/1
-        , exist_in_dictionary/2
+        , exist/2
```

Comment:
This function should be called either `member` or `exists`. Maybe even `includes` or `defines`.